can/broadcastmanager.py

```diff
@@ -290,11 +289,12 @@ def start(self) -> None:
 
     def _run(self) -> None:
         msg_index = 0
-        msg_due_time_ns = time.perf_counter_ns()
 
         if USE_WINDOWS_EVENTS:
             # Make sure the timer is non-signaled before entering the loop
             win32event.WaitForSingleObject(self.event.handle, 0)
+        else:
+            msg_due_time_ns = time.perf_counter_ns()
```

Comment:
Could you move this out of the else clause? Otherwise the IDE complains, that `msg_due_time_ns` might be referenced before assignment. I know that can't actually happen..